src/distributors/local-installer.ts

```diff
@@ -0,0 +1,63 @@
+import * as tc from '@actions/tool-cache';
+
+import fs from 'fs';
+import path from 'path';
+import semver from 'semver';
+import { macOSJavaContentDir, extractJdkFile } from '../util';
+
+import { JavaBase } from './base-installer';
+import { JavaInstallerOptions, JavaDownloadRelease, JavaInstallerResults } from './base-models';
+
+export class LocalDistributor extends JavaBase {
+  constructor(installerOptions: JavaInstallerOptions, private jdkFile?: string) {
+    super('LocalJDKFile', installerOptions);
+  }
+
+  public async setupJava(): Promise<JavaInstallerResults> {
+    let foundJava = this.findInToolcache();
+
+    if (!foundJava) {
```

Comment:
Could you please extend condition with core.info:
```
if (foundJava) {
      core.info(`Resolved Java ${foundJava.javaVersion} from tool-cache`);
} else {
      core.info(`Java ${this.version.raw} is not found in tool-cache. Trying to download...`);
```

Reply:
Done. Thank you.